tests/cross/src/test/scala/tests/pc/CompletionCaseSuite.scala

```diff
@@ -608,4 +608,15 @@ class CompletionCaseSuite extends BaseCompletionSuite {
        |case A.C =>""".stripMargin,
   )
 
+  check(
+    "same-line",
+    """
+      |object A {
+      |  Option(1) match {
+      |    case Some(a) => cas@@
+      |  }
```

Comment:
Is it possible to exclude case completions if there's newline like

```scala
Option(1) match {
  case Some(a) =>
    cas@@
}
```

Reply:
> Instead of checking if they are in the same line, we can check if they start in same column.

Don't need to do that, I think it's too hacky and it may not work with tab as you mentioned :)